core/Test/Framework/Runners/Console.hs

```diff
@@ -24,16 +22,7 @@ import System.Exit
 import System.IO
 
 import Data.Monoid
-
-#if !MIN_VERSION_base(4,7,0)
-instance Functor OptDescr where
-    fmap f (Option a b arg_descr c) = Option a b (fmap f arg_descr) c
-
-instance Functor ArgDescr where
-    fmap f (NoArg a) = NoArg (f a)
-    fmap f (ReqArg g s) = ReqArg (f . g) s
-    fmap f (OptArg g s) = OptArg (f . g) s
-#endif
+import Data.Orphans ()
```

Comment:
The idea of base-orphans is to not require CPP in user code.

Sent from my iPhone

> On 8 May 2015, at 2:50 pm, Herbert Valerio Riedel <notifications@github.com> wrote:
> 
> In core/Test/Framework/Runners/Console.hs:
> 
> > @@ -24,16 +22,7 @@ import System.Exit
> >  import System.IO
> >  
> >  import Data.Monoid
> > -
> > -#if !MIN_VERSION_base(4,7,0)
> > -instance Functor OptDescr where
> > -    fmap f (Option a b arg_descr c) = Option a b (fmap f arg_descr) c
> > -
> > -instance Functor ArgDescr where
> > -    fmap f (NoArg a) = NoArg (f a)
> > -    fmap f (ReqArg g s) = ReqArg (f . g) s
> > -    fmap f (OptArg g s) = OptArg (f . g) s
> > -#endif
> > +import Data.Orphans ()
> this should be MIN_VERSION_base() protected
> 
> —
> Reply to this email directly or view it on GitHub.
>